MapboxNavigation/MapboxSpeechSynthesizer.swift

```diff
@@ -53,7 +59,7 @@ open class MapboxSpeechSynthesizer: NSObject, SpeechSynthesizing {
             hostString = url.host
         }
         
-        self.speech = SpeechSynthesizer(accessToken: accessToken, host: hostString)
```

Comment:
Also while you’re here, can you remove the `_` from the initializer? The `accessToken` argument should be labeled, as with most initializer arguments. Some goes with the MultiplexedSpeechSynthesizer and MapboxSpeechSynthesizer initializers. Thanks!